tony-core/src/main/java/com/linkedin/tony/TaskExecutor.java

```diff
@@ -64,6 +65,10 @@
   private final ScheduledExecutorService hbExec = Executors.newScheduledThreadPool(1);
   private int numFailedHBAttempts = 0;
 
+  private String dockerImagePath;
```

Comment:
I think calling this a "Path" is confusing. Based on my read of https://docs.docker.com/engine/reference/commandline/pull/#concurrent-downloads, seems like it's just the name of an image. Could we just call this `dockerImage` or `dockerImageName`?